### ISSUE TITLE
[a11y-defect-3]: The "sorry, we couldn't find any eligible issues" alert should not appear if a user has deleted their issues

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#61365]
## Point of contact

@joshkimux

## Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

## Details

When a user first comes to the contestable issues page with no pre-loaded issues, they'll receive an alert that says, "Sorry, we couldn't find any eligible issues." This is great! Focus work as expected, and we're informing the users as to why they're seeing nothing here ✔️ 

However, this alert should probably not appear if the user has explicitly added and then subsequently deleted those issues as the user made explicit choices to remove them. Currently, it does appear, and conflicts with focus management on the page which should go to and announce "Add new issues."

https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/96691594-44be-4996-89a2-db47e5fd15cb

## Reproduction steps

1. Log in to NoD as user 236
2. Turn on VoiceOver on Safari
3. Reach the contestable issues page
4. Add one issue
5. Return to the contestable issues page
6. Delete the issue
7. Confirm focus goes to "add a new issues" (correct) but the "sorry, we couldn't find any eligible issues" alert appears and interrupts the focus announcement.

## Proposed solution or next steps

Do not provide the "sorry, we couldn't..." alert if the user has deleted the issues. Unless there is a design decision behind it, we should not show an error on system status "the system couldn't find issues" if the user explicitly chose to do so.

### Next steps
- [x] Confirm with the designer if this behavior is as intended
- [x] If there is a design decision for this, close this ticket out
- [x] If there is no design decision for this, do not show the alert if the user deleted issues they added

## Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [X] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by saderagsdale** [comment]
Tasks:
- Add logic to not show this error for users that explicitly delete issues from their form.

**Comment by joshkimux** [comment]
<img width="778" alt="Screenshot 2023-08-23 at 10 05 43 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/20e350b0-ee77-4d6b-98e2-1c668b1b24a8">
Confirming this has been fixed, perfection 🙌
